src/pudl/transform/ferc1.py

```diff
@@ -1353,6 +1353,23 @@ def rename_calculation_components(calc: str) -> str:
             )
         ).astype({"row_type_xbrl": pd.StringDtype(), "calculations": pd.StringDtype()})
 
+        # Create metadata records for the calculation correction factoids
+        correction_meta = (
```

Comment:
Blanket addition of correction factoids for all calculated values, which may or may not end up getting used -- it'll depend on whether the calculations add up correctly.